tutorials/getting-started/options.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="button.css">
+  </head>
+  <body>
+    <div id="buttonDiv">
```

Comment:
I get that this is where buttons _go_, but:

- the var is called `page` in the JS
- can we call this `id="buttonHolder"` or similar?